packages/@aws-cdk/aws-ecs-patterns/test/fargate/scheduled-fargate-task.test.ts

```diff
@@ -92,6 +92,7 @@ test('Can create a scheduled Fargate Task - with optional props', () => {
       image: ecs.ContainerImage.fromRegistry('henk'),
       memoryLimitMiB: 512,
       cpu: 2,
+      ephemeralStorageGiB: 100,
```

Comment:
Same as above: Let's add in validation failure cases for below 21 and above 200.